molecule/cookiecutter/scenario/driver/digitalocean/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/prepare.yml

```diff
@@ -0,0 +1,11 @@
+---
+{% raw -%}
+- name: Prepare
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Install python for Ansible
+      raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal python-zipstream)
```

Comment:
What's `python-zipstream` for?

Reply:
If I recall I used the aws/gce driver files as a template, and must have left this in.  It's not required, and I've removed it now 👍